src/Build.UnitTests/TargetsFile_Test.cs

```diff
@@ -135,13 +136,13 @@ public void ExplicitAnyCPULibraryProjectIsNot32BitPreferred()
 
                 Project project = ObjectModelHelpers.CreateInMemoryProject(
                     @"
-                   <Project DefaultTargets=`Build` ToolsVersion=`4.0` xmlns=`msbuildnamespace`>
+                   <Project DefaultTargets=`Build` ToolsVersion=`4.8` xmlns=`msbuildnamespace`>
```

Comment:
There is no `ToolsVersion="4.8"`. By the time .NET 4.8 had rolled around we had stopped bumping our version with the .NET version and used the VS version instead. Please either roll these back or delete ToolsVersion entirely.